changelog/fragments/1745609163-journalctl-on-all-docker-variants.yaml

```
@@ -0,0 +1,33 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: bug-fix
+
+# Change summary; a 80ish characters long description of the change.
+summary: |
+  Ship journalctl on the following Docker images variants: basic, complete, ubi and otel collector
```

```suggestion
  Ship journalctl in the elastic-agent, elastic-agent-complete, and elastic-otel-collector UBI based Docker images to enable reading journald logs. It is omitted from elastic-agent-slim and all Wolfi based images.
```

Basic isn't a user facing container name, also it's clearer to just name the containers directly and include where we didn't include it.